tools/metadata_merger/metadata_merger.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python
+
+import argparse
+import sys
+
+import pandas as pd
+
+
+parser = argparse.ArgumentParser()
+parser.add_argument("--dataframe", help="Features table")
+parser.add_argument("--extension", help="File extension of features table")
+parser.add_argument("--metadata", help="Metadata table")
+parser.add_argument("--output", help="Concatenated dataset")
+args = parser.parse_args()
+
+
+def main():
+    # Load features data
+    if args.extension == "tsv" or args.extension == "tabular":
+        dataframe = pd.read_csv(args.dataframe, sep="\t", index_col=0)
+        dataframe = dataframe.transpose()
+    elif args.extension == "csv":
+        dataframe = pd.read_csv(args.dataframe, sep=",", index_col=0)
+    else:
+        print("Wrong format. Input has to be \"*.tsv\" or \"*.csv\" file.", file=sys.stderr)
+        return(1)
+
+    n = len(dataframe)
+    print("Reading features table of {} samples...".format(n), file=sys.stdout)
```

Comment:
You're on python3, check out [f-strings](https://realpython.com/python-f-strings/#f-strings-a-new-and-improved-way-to-format-strings-in-python) for easier formatting.

Reply:
`f"Reading features table of {len(dataframe)} samples..."`

Reply:
That looks much better, somehow never saw that type of formatting.